### ISSUE TITLE
Recursive upload doesn't output a download url

### ISSUE THREAD

**Issue by Fil** [predat/wetransferpy#2]
```
isotype:~ fil$ ls Mars
planeteMars.jpg

isotype:~ fil$ wetransfer-upload -R Mars
Upload file : /Users/fil/Mars/planeteMars.jpg
[ ======================================== ] 101.30%

isotype:~ fil$ 
```

**Comment by predat** [comment]
This python module is initially used with a wetransfer account that's why this fonctionnnality is not really implemented. When you upload a directory you will have a link for each files... I don't think it's what you want.
Perhaps it's better for you to compress that directory first and upload the resulting file.

**Comment by Fil** [comment]
Yes I figured that out — but I wanted to report it as a bug somehow

Thanks a lot for the tool BTW, very useful.
